RetrospectiveExtension.Frontend/components/feedbackItem.tsx

```diff
@@ -24,7 +24,7 @@ import { getUserIdentity } from '../utilities/userIdentityHelper';
 export interface IFeedbackItemProps {
   id: string;
   title: string;
-  columnProps: FeedbackColumnProps;
+  columnProps:FeedbackColumnProps;
```

Comment:
Nit: space here removed on purpose?

Reply:
yeah 🤦 just from some earlier experiment I was doing in this branch, I've removed the formatting changes.